patroni/postgresql/__init__.py

```diff
@@ -745,17 +745,19 @@ def start(self, timeout: Optional[float] = None, task: Optional[CriticalTask] =
         # patroni.
         self.connection_pool.close()
 
+        state = 'starting after custom bootstrap' if self.bootstrap.running_custom_bootstrap else 'starting'
```

Comment:
You also need to change it here: https://github.com/patroni/patroni/blob/c433a4008f41276f4747beabe3450c1a1331fbe7/patroni/postgresql/__init__.py#L709-L710